objects/src/builder/mod.rs

```diff
@@ -267,3 +270,78 @@ impl<T: Rng> AccountBuilder<T> {
         Ok(Account::new(account_id, vault, storage, account_code, self.nonce))
     }
 }
+
+pub struct NoteBuilder {
+    sender: AccountId,
+    inputs: Vec<Felt>,
+    assets: Vec<Asset>,
+    serial_num: Word,
+    tag: Felt,
+    code: String,
+    proof: Option<NoteInclusionProof>,
+}
+
+const DEFAULT_NOTE_CODE: &str = "\
+begin
+end
+";
+
+impl NoteBuilder {
```

Comment:
Thought for the future: once we have a couple of standard scripts (i.e., `P2ID` and `P2IDR`) it might be good to create specialized builders specifically for these scripts. For example, something like:
```Rust
pub fn build_p2id(sender: AccountId, target: AccountId, assets: Vec<Asset>) -> Note
```